src/bin/zmlicensectl

```diff
@@ -64,10 +64,15 @@ startLicenseDaemonService()
 	if [ "x$zimbra_license_daemon_offline_mode" = "x" ]; then
 		zimbra_license_daemon_offline_mode=false
 	fi
+	if [ "x$zimbra_license_proxy_timeout" = "x" ]; then
```

Comment:
Lets make more generic..
prefix all needed props with say: `ext|external.{propertyName}` and use that prefix here to pull all such prefixed prop and pass them to the service startup command.. will be useful to expose more properties in future.

Reply:
https://github.com/Zimbra/zm-license-daemon-service/pull/88#discussion_r1908628256